pages/documentation/simulation/powerflow/index.md

```diff
@@ -170,21 +170,22 @@ minimum or maximum: this corresponds to a valid load flow result for the ratio t
 ## Implementations
 
 The following power flow implementations are supported:
-- [Open LoadFlow](openlf.md)
-- [Hades2](https://rte-france.github.io/hades2/index.html): learn how to
-install Hades2 and use it with PowSyBl.
+- [PowSyBl OpenLoadFlow](openlf.md)
+- [Hades2](hades2.md)
 
 ## Configuration
 You first need to choose which implementation to use in your configuration file:
 ```yaml
 load-flow:
-  default-impl-name: "hades2"
-```
-or
-```yaml
-load-flow:
-  default-impl-name: "OpenLoadFlow"
+  default-impl-name: "<IMPLEMENTATION_NAME>"
 ```
+
+Each implementation is identified by its name, that may be unique in the classpath:
+- use "OpenLoadFlow" to use PowSyBl OpenLoadFlow
+- use "Hades2" to use RTE Hades2
```

Comment:
There is no capital h at the beginning if you are talking about the default-impl-name
```suggestion
- use "hades2" to use RTE Hades2
```

Reply:
In powsybl implementation, the name is indeed with a capital h in our latest release version. Maybe it should not?

Reply:
Ok I did not know it had changed